test/augmentation/test_augmentation.py

```diff
@@ -16,11 +17,15 @@
     LongestMaxSize,
     Normalize,
     PadTo,
+    RandomAffine,
```

Comment:
maybe is missing some tests for these augmentations too? @shijianjian

Reply:
Probably. I was thinking that it might be better to have a smoke tester by:
```python
    for name, obj in inspect.getmembers(sys.modules["kornia.augmentation"]):
        if inspect.isclass(obj) and isinstance(obj, AugmentationBase2D):
            ...
```

Reply:
I don't think this can be automated in this way granted a number of these augmentations don't have a default value set in their init method. This is why I manually passed some init values in my test to the augmentations I am testing.

But I guess I could manually add all the remaining augmentations for testing (what a chore 😅) if you think this is relevant. 
For what it's worth, I have only seen the undesired behavior in transforms that use the PRNG, but then we are indeed never safe from the issue popping up elsewhere.

What do you think?

Reply:
Sure, but let's add it into a separate PR @NielsPichon  -- we have an issue tracker for the missing tests #2231